src/services/depositService.ts

```diff
@@ -141,6 +140,30 @@ export function createDepositEVMERC20Transaction(
   }
 }
 
+export function createDepositSiloToSiloTransaction(
```

Comment:
nit: it should be `createDepositFromSiloTransaction`. Yes, it might use SiloToSilo contract, but it might not.